base/server/etc/default.cfg

```diff
@@ -663,3 +664,27 @@ pki_registry_enable=False
 # - acme_realm_groups_dn
 #
 # See /usr/share/pki/acme/realm/<type>/realm.conf
+[EST]
+pki_ds_setup=False
+pki_security_domain_setup=False
+pki_registry_enable=False
+pki_ca_uri=https://%(pki_hostname)s:%(pki_https_port)s
+est_ca_profile=estServiceCert
+est_ca_user_name=
+est_ca_user_password=
+est_ca_user_password_file=
+est_ca_user_certificate=
+est_realm_type=
+est_realm_custom=
+est_realm_url=
+est_realm_auth_type=BasicAuth
+est_realm_bind_dn=cn=Directory Manager
+est_realm_bind_password=
+est_realm_nickname=
+est_realm_user=
+est_realm_username=
+est_realm_password=
+est_realm_users_dn=ou=people,dc=est,dc=pki,dc=example,dc=com
+est_realm_groups_dn=ou=groups,dc=est,dc=pki,dc=example,dc=com
+est_realm_statements=/usr/share/pki/est/conf/realm/statements.conf
+est_authorizer_exec_path=/usr/libexec/estauthz
```

Comment:
The path in `authorizer.conf` is different.

BTW, the `estauthz` script looks more like a sample script instead of a general purpose tool. Would it be better to install it somewhere under `/usr/share/pki/est` instead of `/usr/libexec`?

Reply:
Moved the script to `/usr/share/pki/est/bin/estauthz` and update all the references.